code/script-math.js

```diff
@@ -0,0 +1,56 @@
+var divisionPerformed = false;
+
+// displays numerous inputs (numbers) on the screen
+function displayNumber(inputValue) {
+  nameCalculator.nameDisplayMain.value = nameCalculator.nameDisplayMain.value + inputValue;
+}
+
+// testing 0 input, when 0 comes after / buttons are disabled and message is given
+function testZero(inputValue) {
+  if (divisionPerformed == true && inputValue == 0) {
+    var rCalcButton = document.getElementsByClassName("rcalc-button");
```

Comment:
rCalc prefix use just for css ... here you use ..
button / buttonForDisable ...etc

Reply:
var rCalcButton updated to var buttonDisable when it disables buttons and var button Enable is used when buttons go enable